app/models/observation_accuracy_validator.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class ObservationAccuracyValidator < ApplicationRecord
+  belongs_to :user
+  has_and_belongs_to_many :observation_accuracy_samples
+end
```

Comment:
if you do add `observation_accuracy_experiment_id` to `observation_accuracy_validators`, it would be good to add a uniqueness validator here, e.g. `validates_uniqueness_of :user_id, scope: :observation_accuracy_experiment_id`